tests/mint.test.js

```diff
@@ -0,0 +1,62 @@
+require('dotenv').config()
+const shell = require('shelljs')
+const util = require('../lib/algoUtil')
+const {EncodeUint, EncodeBytes} = util
+const algosdk = require('algosdk')
+
+const server = "http://127.0.0.1"
+const port = 8080
+
+var adminAccount, receiverAccount, token, clientV2, appId
+
+beforeEach(async () => {
+  await privateTestNetSetup(appId)
+  adminAccount = accounts[0]
+  receiverAccount = accounts[1]
+
+  token = await shell.cat(`devnet/Primary/algod.token`).stdout
+  clientV2 = new algosdk.Algodv2(token, server, port)
+  let info = await util.deploySecurityToken(clientV2, adminAccount)
+  appId = info.appId
+
+  //opt in
+  await util.optInApp(clientV2, receiverAccount, appId)
+})
+
+// describe('when receiver max balance is not set', async () => {
```

Comment:
Probably should remove the commented out describe blocks or add them back in.

Reply:
Yep, as I figure out later describe blocks don't work with async code in Jest.
Let me remove that.